internal/service/handlers/daily_questions_status.go

```diff
@@ -0,0 +1,133 @@
+package handlers
+
+import (
+	"fmt"
+	"math"
+	"net/http"
+	"time"
+
+	"github.com/rarimo/geo-points-svc/internal/data"
+	"github.com/rarimo/geo-points-svc/internal/data/evtypes/models"
+	"github.com/rarimo/geo-points-svc/internal/service/requests"
+	"github.com/rarimo/geo-points-svc/resources"
+	"gitlab.com/distributed_lab/ape"
+	"gitlab.com/distributed_lab/ape/problems"
+)
+
+func GetDailyQuestionsStatus(w http.ResponseWriter, r *http.Request) {
+	var AlreadyDoneForUser bool
+	var TimeToNext int64
+
+	req, err := requests.NewGetDailyQuestionsRequest(r)
+	if err != nil {
+		Log(r).Errorf("error getting status daily questions: %v", err)
+		ape.RenderErr(w, problems.InternalError())
+		return
+	}
+
+	// Закомментировано, т.к. аутентификация временно отключена
+	// if !auth.Authenticates(UserClaims(r), auth.UserGrant(req.Nullifier)) {
+	// 	ape.RenderErr(w, problems.Unauthorized())
+	// 	return
+	// }
+
+	currentTime := time.Now().UTC()
+
+	activeQuestions, err := DailyQuestionsQ(r).FilterActive().Select()
+	if err != nil {
+		Log(r).Errorf("error getting active questions: %v", err)
+		ape.RenderErr(w, problems.InternalError())
+		return
+	}
+	Log(r).Infof("activeQuestions: %v", len(activeQuestions))
```

Comment:
Time zone from config

You need to get last user question id and select not included this id (`FilterID`).
```suggestion
    var localTime time.Time
    if timeZone < 0 {
        year, month, day := time.Now().UTC().Sub((-timeZone) * time.Hour).Date()
        localTime = time.Date(year, month, day, 0, 0, 0, 0, time.UTC)
    } else {
        year, month, day := time.Now().UTC().Add(timeZone * time.Hour).Date()
        localTime = time.Date(year, month, day, 0, 0, 0, 0, time.UTC)
    }
    
	activeQuestion, err := DailyQuestionsQ(r).FilterByStartsAtAfter(localTime).OrderByStartsAt("ASC").Get()
	if err != nil {
		Log(r).WithError(err).Error("Failed to get active questions")
		ape.RenderErr(w, problems.InternalError())
		return
	}
	Log(r).Debugf("Active questions: %+v", len(activeQuestions))
```